server/src/main/java/ug/edu/pl/server/domain/group/Member.java

```diff
@@ -0,0 +1,34 @@
+package ug.edu.pl.server.domain.group;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.DynamicUpdate;
+import ug.edu.pl.server.domain.common.persistance.BaseEntity;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@DynamicUpdate
+@Entity
+@Table(name = "members")
+class Member extends BaseEntity {
+    @Column(nullable = false)
+    private Long userId;
+
+    @Column(nullable = false)
+    private String nickname;
+
+    @OneToOne(optional = false)
```

Comment:
I don't think that user - role is a 1-to-1 relationship, rather either n-to-1 or n-to-m